### PR TITLE
NOISSUE - Allow termination of manager

### PR DESCRIPTION
**Pull request by SammyOina**
# What type of PR is this?

This is a feature because it adds manager client termination.

## What does this do?

Allows server to send termination requests to clients
Provides server with authentication information used by server

## Which issue(s) does this PR fix/relate to?

- None

## Have you included tests for your changes?

No, I have tested manually

## Did you document any new/modified feature?

No, I have not updated the documentation because no change in outward functionality.

### Notes
to be merged after #94

### CODE REVIEW

**Comment by drasko** [comment]
I would need @dborovcanin and @arvindh123 to review his one

File: .github/workflows/checkproto.yaml [hunk 1]
```diff
@@ -55,7 +55,7 @@ jobs:
       - name: Set up Cocos-AI
         run: |
           # Rename .pb.go files to .pb.go.tmp to prevent conflicts
-          for p in $(ls manager/*.pb.go); do
+          for p in $(ls pkg/manager/*.pb.go); do
```
Comment (drasko) [inline .github/workflows/checkproto.yaml:58]: I do not like that this in `pkg` is called `manager`, like other `manager` in the top level. It is confusing.

**Review by drasko — CHANGES_REQUESTED** [review]

File: test/computations/main.go [hunk 2]
```diff
@@ -53,17 +54,21 @@ func (s *svc) Run(ipAdress string, reqChan chan *manager.ComputationRunReq) {
...
-		ResultConsumers: []string{"consumer1"},
-		AgentConfig: &manager.AgentConfig{
-			Port:        "7002",
-			LogLevel:    "debug",
-			AttestedTls: attestedTLS,
+	reqChan <- &manager.ServerStreamMessage{
+		Message: &manager.ServerStreamMessage_RunReq{
+			RunReq: &manager.ComputationRunReq{
+				Id:              "1",
+				Name:            "sample computation",
```
Comment (drasko) [inline test/computations/main.go:61]: What is this naming (`sample ...`)? Looks like some test, but it is actually in `main()`

**Review by drasko — CHANGES_REQUESTED** [review]

File: test/computations/main.go (hunk 2 above)
Comment (SammyOina) [reply test/computations/main.go:61]: this is the test server which we use as an example for users as a standin for backends service

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**